src/lrmaCU/terra/workspace_utils.py

```diff
@@ -22,6 +22,18 @@ def get_workspace_bucket(ns: str, ws: str, max_attempts: int = 2) -> str:
     return _query_workspace(ns, ws, max_attempts)['bucketName']
 
 
+def get_workspace_attributes(ns: str, ws: str, max_attempts: int = 2) -> dict[str, str]:
```

Comment:
I'd suggest rename this function a bit (sorry for naming the existing one sub-optimally).

`get_workspace_all_attributes` to avoid (easy to make) typos